src/main/java/com/hubspot/jinjava/lib/filter/UnionFilter.java

```diff
@@ -0,0 +1,53 @@
+package com.hubspot.jinjava.lib.filter;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Map;
+
+import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
+import com.hubspot.jinjava.doc.annotations.JinjavaParam;
+import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
+import com.hubspot.jinjava.util.ForLoop;
+import com.hubspot.jinjava.util.ObjectIterator;
+
+@JinjavaDoc(
+    value = "Returns a list containing elements present in either list",
+    params = {
+        @JinjavaParam(value = "value", type = "sequence", desc = "The first list"),
+        @JinjavaParam(value = "list", type = "sequence", desc = "The second list")
+    },
+    snippets = {
+        @JinjavaSnippet(
+            code = "{{ [1, 2, 3]|union([2, 3, 4]) }}")
+    })
+public class UnionFilter implements AdvancedFilter {
+
+  @Override
+  public Object filter(Object var, JinjavaInterpreter interpreter, Object[] args, Map<String, Object> kwargs) {
+
+    LinkedHashSet<Object> result = new LinkedHashSet<>();
```

Comment:
How about using `Sets.union`?

Reply:
The reason I did not use the set function helpers is that they would be slower/require more space. I would need to loop over the two iterables to form the sets, then apply those functions. This wouldn't be noticeable for small sets but might be for large sets. The functions are pretty simple anyways, and I'm not using the set helper functions is worth it but it's up to you.